src/util/compatibility/qatomic.h

```diff
@@ -9,24 +9,9 @@
 // Until the minimum required Qt version of Mixxx is increased some utility
 // functions that work independently of the Qt version are needed.
 
-template<typename T>
-inline T atomicLoadAcquire(const QAtomicInteger<T>& atomicInt) {
-#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
-    return atomicInt.loadAcquire();
-#else
-    return atomicInt.load();
-#endif
-}
-
-template<typename T>
-inline T* atomicLoadAcquire(const QAtomicPointer<T>& atomicPtr) {
-#if QT_VERSION >= QT_VERSION_CHECK(5, 14, 0)
-    return atomicPtr.loadAcquire();
-#else
-    return atomicPtr.load();
-#endif
-}
+#define DEPRECATED_QATOMIC_OPS /// Deprecated, use the corresponding member function directly
 
+DEPRECATED_QATOMIC_OPS
```

Comment:
What does this do? 
Did you consider to fix the code instead?

Reply:
It just copy-pastes the message above so it's shown during code completion. I could fix this (since `main` is Qt6 only), but that would be touching another 20 files which I tried to avoid. I can do it though if you prefer.

Reply:
The preprocessor stops at comments. In my case Eclipse shows as macro expansion just an empty field. 
So I think you should remove this and the comment all together. From the function content it is clear that the function is deprecated, because the main branch is on QT6.  

It is OK to keep the current state with this wrapper since there is no user benefit to remove it.

Reply:
I have removed the comment and `#define`